index.js

```diff
@@ -1,8 +1,9 @@
+const FileType = import('file-type');
```

Comment:
cc @staltz

Reply:
What was the motivation to upgrading `file-type`? I would prefer to stick to commonJS as long as possible. ESM adds too many headaches. And no, noderify doesn't support it yet, but esbuild is a suitable replacement to noderify that supports ESM.

But still, I would rather avoid all this work if we're upgrading `file-type` just for upgrading sake.

Reply:
The version of file-type we are running is throwing npm audit warnings because of some vulnerable dep
Bonus woulo be that updating gets us newer support / fixes.

Yeah esm is real annoying